ale/base/data_naif.py

```diff
@@ -150,10 +151,22 @@ def sensor_position(self):
 
     @property
     def frame_chain(self):
-        pass
-
-    @property
-    def _sensor_orientation(self):
+        # pass
+        if not hasattr(self, '_frame_chain'):
+            frame_chain = {}
+            frame_chain['j2000'] = FrameNode(1)
+            # self._frame_chain['spacecraft'] = FrameNode(self.)
+            frame_chain['sensor'] = FrameNode(self.sensor_frame_id, 
+                                                parent=frame_chain['j2000'],
+                                                rotation='ConstantRotation')
```

Comment:
This should be an actual ConstantRotation or TimeDependentRotation object, see rotation.py. You can use the [pxform](https://naif.jpl.nasa.gov/pub/naif/toolkit_docs/FORTRAN/spicelib/pxform.html) function to get the rotation matrix.

Reply:
Just to make sure I am fully understanding this: this rotation matrix should be able to be passed straight into the construction of a rotation object as the "quat" value, right?